Ginger/GingerCoreNETUnitTest/RunTestslib/ActReturnValueTest.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Amdocs.Ginger.Repository;
+using Amdocs.Ginger.Repository;
+using Ginger.Run;
+using GingerCore;
+using GingerCore.Actions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GingerCoreNETUnitTest.RunTestslib
+{
+    [TestClass]
+    public class ActReturnValueTest
+    {
+        static ActDummy act;
```

Comment:
Please create a separate act for each unit test.
Using same object may create issue when running tests in parallel